src/store/concepts/movieCatalogs/selectors.js

```diff
@@ -0,0 +1,42 @@
+import { createSelector } from 'reselect'
+import { path } from 'ramda'
+import { map, pick } from 'lodash'
+
+const movieDataSelector = state => (
+  path(['data', 'movies'], state)
+)
+
+const movieCatalogEntriesSelector = (state, endpoint) => (
+  path(['movieCatalogs', endpoint, 'entries'], state)
```

Comment:
do you still perform unnecessary sorting by endpoint?

Reply:
removed